cellfinder/__init__.py

```diff
@@ -1,3 +1,10 @@
+import warnings
+
+warnings.warn(
+    "cellfinder (CLI) has migrated. Please use brainglobe-workflows instead: https://github.com/brainglobe/brainglobe-workflows",
```

Comment:
```suggestion
    "cellfinder (the command line tool) has migrated. Please use brainglobe-workflows instead: https://github.com/brainglobe/brainglobe-workflows",
```
Just because some users may not know what a CLI is? Just a thought.